.github/workflows/ci.yml

```
@@ -53,6 +53,14 @@ jobs:
         if: github.event_name != 'release' && github.event_name != 'workflow_dispatch'
       - run: just env-info
       - run: just test-fmt
+      - run: npm install
+        working-directory: martin/martin-ui
+      - name: Run martin/martin-ui linting
+        run: just biomejs-martin-ui
+      - name: Run martin/martin-ui tests
+        run: just test-frontend
+      - name: Run martin/martin-ui type checks
+        run: just type-check
```

Not sure we need to document each step.  In the future, all these will be moved to a single `just` command anyway

```suggestion
      - run: just biomejs-martin-ui
      - run: just test-frontend
      - run: just type-check
```

---

Totally agree, but let's do things one step at a time